cbits/phone-numbers.cc

```diff
@@ -94,3 +94,15 @@ extern "C" bool _c_phone_number_util_is_valid_number(
   return ((PhoneNumberUtil *)util_instance)
       ->IsValidNumber(*((PhoneNumber *)phone_no));
 }
+
+extern "C" int _c_phone_number_get_country_code_source(
```

Comment:
While explicit conversion is more robust, the original code does not do it either, it just assumes the enums defined here are exactly the same as libphonenumber. Better be consistent.

Reply:
So what I was thinking here, is that the other enums have identical [c++ counterparts](https://github.com/google/libphonenumber/blob/f52af239997c35335894206a4a45f44558ee71a0/java/libphonenumber/src/com/google/i18n/phonenumbers/PhoneNumberUtil.java#L405).

Whereas the `CountryCodeSource` is from a protobuf, and has some [odd values specified](https://github.com/google/libphonenumber/blob/0dd0f034ff89d00f4d67cee8d8b88b27be4931bd/resources/phonenumber.proto#L106).

So I think the manual conversion is necessary

Reply:
Ok fair! Perhaps add a comment to clarify the source of this enum (`phonenumber.pb.h`), because at the beginning of the file, it says enums come from `phonenumberutil.h`.